### PR TITLE
quic: emit process warning on --tls-trace

### PR DESCRIPTION
**Pull request by jasnell**

### CODE REVIEW

File: src/quic/node_quic_crypto.cc [hunk 1]
```diff
@@ -650,17 +651,24 @@ void SetHostname(SSL* ssl, const std::string& hostname) {
 }  // namespace
 
 void InitializeTLS(QuicSession* session) {
+  static bool warn_trace_tls = true;
```
Comment (addaleax) [inline src/quic/node_quic_crypto.cc]: This should not be global mutable state, and stored on the `Environment` like we do for other warnings we only want to emit once

**Comment by jasnell** [comment]
Fixed!

File: src/quic/node_quic_crypto.cc [hunk 2]
```diff
@@ -631,16 +632,22 @@ void SetHostname(SSL* ssl, const std::string& hostname) {
 
 void InitializeTLS(QuicSession* session) {
   QuicCryptoContext* ctx = session->CryptoContext();
-
+  Environment* env = session->env();
   SSL* ssl = ctx->ssl();
   SSL_set_app_data(ssl, session);
   SSL_set_cert_cb(ssl, CertCB, session);
   SSL_set_verify(ssl, SSL_VERIFY_NONE, crypto::VerifyCallback);
+  Environment* env = session->env();
```
Comment (addaleax) [inline src/quic/node_quic_crypto.cc]: Is this being added twice now?

```suggestion
```

**Comment by jasnell** [comment]
Landed

**Pull request closed without merging**